### ISSUE TITLE
v0.15.1 Quest notes: Inventor - Inevitability of Rust

### ISSUE THREAD

**Issue by TerryCavanagh** [TerryCavanagh/diceydungeons.com#784]
Inevitability of Rust suffers a little bit from clarity - it can be a little hard to follow what's going on as you use your equipment.

Two mechanics in this quest worry me a little bit:
 - Upgrading equipment to fix it's durability
 - Using Grindstone to protect important equipment (and grindstone in general, as a quest specific crutch)

Try removing both of these elements, and seeing if it's an improvement. If it is, rebalance around that.

**Comment by TerryCavanagh** [comment]
Thanks for the thoughts!

Yeah, might be worth just removing reusable equipment from this quest completely. Which will be a little awkward for things like Spanner and Bump, which get extra uses on upgrade, but probably something I can work around with substitutions.

The subtly cool thing going on in this quest is that all equipment is reusable, and I'm wondering if there's more I can do to explore that.

In two minds about Grindstone. On the one hand I think it's fun and leads to some interesting decisions, on the other, I think it sort of becomes the focus of the quest, and I wonder if it might be stronger without it. Not sure yet.

Grindstone actually allows things to go beyond six! The durability counter just adds an extra dice beside it:

![image](https://user-images.githubusercontent.com/2874150/51086702-c364ff80-1741-11e9-8d5f-f81beb3eb395.png)

**Comment by YinYin-Falcon** [comment]
Oh, well I didn't get that extra dice. It just stuck at 6 - but perhaps I didn't actually repair beyond.

If you want to explore this mode without grind stone, I'd suggest you make it one of two choices for first level up (like crowbar on thief).

The other one will have to be similarly awesome of course. Perhaps something that capitalises on the other side of the coin: making gadgets when things break. Maybe a card that becomes more powerful with each created gadget.

**Comment by TerryCavanagh** [comment]
Some small bugs to fix here, but I think this is one of the best episodes in the game. Closing!
